tests/test_lookup_table.py

```diff
@@ -1,82 +1,63 @@
 
-
-def test_contention_text_lookup_table_duplicate_entries(mock_file_open):
+def test_contention_text_lookup_table_duplicate_entries() -> None:
     """Test how ContentionTextLookupTable handles duplicate entries."""
     duplicate_csv = (
         "CONTENTION TEXT,CLASSIFICATION CODE,CLASSIFICATION TEXT\n"
         "PTSD,8989,Mental Disorders\n"
         "PTSD,9999,Different Mental Disorder\n"
     )
-    with patch("builtins.open", mock_file_open(read_data=duplicate_csv)):
+    with patch("builtins.open", mock_open(read_data=duplicate_csv)):
         table = ContentionTextLookupTable(init_values=dropdown_expanded_table_inits)
         # Last entry should win
         entry = table.get("PTSD")
         assert entry["classification_code"] == 9999
         assert entry["classification_name"] == "Different Mental Disorder"
 
-
-def test_contention_text_lookup_table(mock_file_open, mock_csv_strings):
+def test_contention_text_lookup_table(mock_csv_strings: Dict[str, str]) -> None:
     """Test ContentionTextLookupTable with mock data."""
-    with patch("builtins.open", mock_file_open(read_data=mock_csv_strings["contention_csv"])):
+    with patch("builtins.open", mock_open(read_data=mock_csv_strings["contention_csv"])):
         table = ContentionTextLookupTable(init_values=dropdown_expanded_table_inits)
         assert table.get("PTSD")["classification_code"] == 8989
         assert table.get("Knee pain")["classification_code"] == 8997
         assert table.get("Unknown")["classification_code"] is None
-
-
-def test_diagnostic_code_lookup_table_file_error():
-    """Test error handling when file cannot be opened."""
-    with patch("builtins.open", side_effect=FileNotFoundError()):
-        with pytest.raises(FileNotFoundError):
-            DiagnosticCodeLookupTable(init_values=diagnostic_code_inits)
-
-
-def test_contention_text_lookup_table_file_error():
-    """Test error handling when file cannot be opened."""
-    with patch("builtins.open", side_effect=FileNotFoundError()):
-        with pytest.raises(FileNotFoundError):
-            ContentionTextLookupTable(init_values=dropdown_expanded_table_inits)
-
-
-def test_diagnostic_code_lookup_table_empty_file(mock_file_open):
-    """Test DiagnosticCodeLookupTable with empty file."""
-    with patch("builtins.open", mock_file_open(read_data="DIAGNOSTIC_CODE,CLASSIFICATION_CODE,CLASSIFICATION_TEXT\n")):
-        table = DiagnosticCodeLookupTable(init_values=diagnostic_code_inits)
-        assert table.get(1234)["classification_name"] is None
-
-
-def test_contention_text_lookup_table_empty_file(mock_file_open):
-    """Test ContentionTextLookupTable with empty file."""
-    with patch("builtins.open", mock_file_open(read_data="CONTENTION TEXT,CLASSIFICATION CODE,CLASSIFICATION TEXT\n")):
-        table = ContentionTextLookupTable(init_values=dropdown_expanded_table_inits)
-        assert table.get("Test")["classification_code"] is None
```

Comment:
These were added when I was attempting to increase test coverage. Are we just removing marginal value tests?

Reply:
https://youtu.be/m0b_D2JgZgY?si=-agsZ8HpCqTrUGBs